docs/source/dev/index.rst

```diff
@@ -103,10 +103,18 @@ generally debugging and building out features.
 Other Documentation
 ~~~~~~~~~~~~~~~~~~~
 Additional documentation exists that may be useful for developers seeking deeper
-understanding of the solver and mathematics.  This documentation is not generally
-necessary for most development efforts.
-
-.. toctree::
-   :maxdepth: 1
-
-   other_docs.rst
+understanding of the solver and mathematics.
+
+- `NWTC Programmer’s Handbook <https://drive.google.com/file/d/1bDV1fBkiZUWs6Tkzb6nhCMUQvHpN_OtM/view?usp=sharing>`_
+   This is an overview of programming guidelines for FAST 8. While some syntax and minor details have
+   changed in OpenFAST, most of this guide is still relevant.
+- :download:`DCM_Interpolation.pdf <../../OtherSupporting/DCM_Interpolation/DCM_Interpolation.pdf>`
```

Comment:
See my comments below regarding the "DCM_Interpolation" and "OpenFAST Algorithms" documents.  They are repeated here and in the new sections 4.3-4.4 of readthedocs.  Suggest putting them in section 4 instead of here.